bin/pidtree-bcc

```diff
@@ -10,4 +10,4 @@ if [ "$INSTALL_HEADERS_AT_RUNTIME" = "true" ]; then
   fi
 fi
 
-/opt/venvs/pidtree-bcc/bin/python3 -m pidtree_bcc.main $@
+exec /opt/venvs/pidtree-bcc/bin/python3 -m pidtree_bcc.main $@
```

Comment:
FWIW I think we could do a bit of an unhack on pidtree at some point to remove some of the vestigial remnants of how it grew from a dockerised hack into an actual program - this file is one of those things that could probably do with a fresh objective viewpoint.